meson.build

```diff
@@ -1,8 +1,10 @@
 project('ubsh', 'c',
   default_options: [
     'warning_level=3',
+    'werror=true',
```

Comment:
I'm not a big fan of Werror, I like to see the difference between actual errors and warnings

Reply:
That's fair. I don't really mind, not using Werror also makes for faster development. I'm just against warnings

Reply:
I'm also not a fan of Werror. I've seen too many project which did not compile because of an update of GCC bringing a new warning and the project used Werror.